pkg/controller/node/node_controller.go

```diff
@@ -163,14 +165,19 @@ func filterSegmentPorts(nsxClients *NsxClients, ports []*data.StructValue, nodeN
 }
 
 func (r *ReconcileNode) createNsxClients() (*NsxClients, error) {
-	// TODO: get configurations from configmap_controller
-	log.Info("Getting NCP configmap for node controller")
-	ncpConfigMap := &corev1.ConfigMap{}
-	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: operatortypes.NsxNamespace, Name: operatortypes.NcpConfigMapName}, ncpConfigMap)
-	if err != nil {
-		return nil, err
+	configMap := r.sharedInfo.OperatorConfigMap
+	if configMap == nil {
+		log.Info("Getting config from operator configmap")
+		configMap = &corev1.ConfigMap{}
+		err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: operatortypes.OperatorNamespace, Name: operatortypes.ConfigMapName}, configMap)
```

Comment:
nit: rename `operatortypes` to `ncptypes` or vice-versa to avoid confusion

Reply:
Sure, I noticed that too. Will unify to operatortypes in a following patch.